tests/slow/src/test/scala/tests/feature/Scala3CodeActionLspSuite.scala

```diff
@@ -12,12 +13,16 @@ import scala.meta.internal.mtags.MtagsEnrichments.XtensionAbsolutePath
 
 import munit.Location
 import munit.TestOptions
+import org.eclipse.lsp4j.CodeAction
 import tests.codeactions.BaseCodeActionLspSuite
 
 class Scala3CodeActionLspSuite
     extends BaseCodeActionLspSuite("cross-code-actions") {
 
   override protected val scalaVersion: String = BuildInfo.scala3
+  val filterAction: CodeAction => Boolean = { act: CodeAction =>
```

Comment:
Why is this needed? I see that we anyway `ConvertToNamedArguments` action showing up below?